tools/integration_tests/local_file/helpers/file_operations_helper.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2023 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package helpers
+
+import (
+	"os"
+	"strings"
+	"testing"
+)
+
+func ValidateNoFileOrDirError(path string, t *testing.T) {
```

Comment:
Please move this to a more generic helper/util outside local_file tests.

Reply:
Sorry, I missed this during review.

Anything not related to GCS shouldn't be in this file: https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/util/client/gcs_helper.go
WritingToLocalFileShouldNotWriteToGCS -> This is specific flow not related to GCS. only operations on GCS has to be in gcs_helper.

You can take it up after all these PRs are merged.

Reply:
moved NewFileShouldGetSyncedToGCSAtClose and WritingToLocalFileShouldNotWriteToGCS helper methods to local_file tests as they were specific to local file tests. Once all these PRs are merged, I will take another look.